api/v1alpha/subschema/common.schema.json

```diff
@@ -58,6 +58,10 @@
     "version": {
       "description": "Which framework version to use. Please consult https://docs.saucelabs.com/web-apps/automated-testing/cypress#supported-testing-platforms for available versions.",
       "type": "string"
+    },
```

Comment:
You also have to generate the schema. Not sure why the `check-schema` pipeline didn't fail :(

Reply:
generated :D